api/billingd/service/service.go

```diff
@@ -987,7 +1000,8 @@ func (s *Service) handleUsage(ctx context.Context, cus *Customer, product Produc
 	}
 	update := bson.M{"daily_usage." + product.Key + ".total": total}
 
-	if total > product.FreeQuotaSize && !cus.Billable {
+	var err error
+	if total > product.FreeQuotaSize && !cus.Billable && cus.ParentKey == "" {
```

Comment:
Never error when checking usage for a child. If their parent is not billable and over a free quota, they will see the error when their usage is accrued on the parent, but it makes no sense to also error here... and was causing the slate issue: the parent account was billable, but the child accounts were still erroring (since they are never billable).